exporter/elasticsearchexporter/README.md

```diff
@@ -118,33 +118,38 @@ Using the common `batcher` functionality provides several benefits over the defa
 
 ### Elasticsearch document routing
 
-Telemetry data will be written to signal specific data streams by default:
-logs to `logs-generic-default`, metrics to `metrics-generic-default`, and traces to `traces-generic-default`.
+Documents are routed to the target index / data stream dynamically in the following order. The first routing mode that applies will be used.
+1. "Static mode": To `logs_index` (for log records), `metrics_index` (for data points) and `traces_index` (for spans and span events) if they are configured.
```

Comment:
Should we use `logs_index` for span events? Or maybe add another option for span events?

Reply:
sgtm. It is explicitly stated that it is not recommended to set `logs_index`, `metrics_index` and `traces_index` anyway.

Reply:
Done in [61a3bfa](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/38500/commits/61a3bfabde57c5de3d74e7df6f9ee6451ba1d2bf)